app/assets/javascripts/component_data/member.js.coffee

```diff
@@ -1,16 +1,15 @@
 @MemberData = flight.component ->
   @after 'initialize', ->
     return if not gon.user
-    channel = @attr.pusher.subscribe("private-#{gon.user.sn}")
 
-    channel.bind 'account', (data) =>
+    @attr.ranger.bind "#{gon.user.sn}.account", (data) =>
```

Comment:
I have not seen such event in peatio code, what do you expect to get from such event?